### DISCUSSION TITLE
typo

### DISCUSSION THREAD

**Discussion by clarkbremer** [SpenceKonde/megaTinyCore#693] [Library discussion]
https://github.com/SpenceKonde/megaTinyCore/blob/master/megaavr/extras/ATtiny_x24.md
Column header for `ATtiny1627` should be `ATtiny1624`.

**Comment by SpenceKonde** [comment]
Thanks will fix. 

A trio of tips below for how you can get more prestige from reporting this kind of thing (i am very certain that my documentation has worse spelling and grammar in some places than the label of a low end import..... but I don't see my own mistakes and most people I've asked refuse to do it, even for money because they say it's too technical, they don't feel like the're qualified to do it. But I just know people who are technical enough to assess are judging me to be a moron because of typos. and  wish to fix them, but am unable to get the help I need. 

First tip
A typo in the documentation is an issue, not a discussion. There's not much to discuss, I forgot to change a number when copy/pasting.
A discussion is better suited for "It seems like this is possible but i can't figure out quite how" or "This behavior is not what I expected, but I may be misunderstanding the intent. Is this a bug or am I confused:) 

Second tip:
Correcting this typo with a pull request would have immortalized you as a contributor to a major project. I understand hesitation about very complicated PR's. but this is a "Hey, want an instant bit of extra respect from everyone reading yoru github history for 2 minutes of work?) Just sayin - these days active GitHub contributions can decide who gets hired for a job. 

Third tip: 
Someone who was very productive at improving spelling,. grammar, and catching sentences that end halfway through or are flat out wrong, and submitted PRs to correct them would be eventually asked to email me their mailing address to me so I could send them some free products - including some stuff that is not for sale to the public. absolutely free.  I want my documentation to be so good, that if it meets the docs for the Official core in a dark alley way, there will be a loud shout,. a scuffle, and only my documentation would leave the alley.

**Reply by clarkbremer** [reply]
Wow, I hope that was a canned response, not written just for me :-)

I'll choose option 2.  Actually. I did it last week, and you already merged it.  I'm retired, so the "extra respect" doesn't matter.  But "immortality"?  Hmmmmm.

I just got some 824 chips, but I'm still waiting on my Schottky diodes.  I'll be replacing an ATMega328p in an existing design.  I will be particularly interested in the sleep modes, including waking up from an external interrupt.  In the existing design, I'm using the `<LowPower.h>` library.  Not sure it that will work with this core.   But I'll keep my eyes open for ways to improve the documentation in that area, at least.

Thanks again for leading this effort.  CB.
